db/migration/1722415645057-AddChartConfigHash.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddChartConfigHash1722415645057 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`
+            ALTER TABLE chart_configs
+            ADD COLUMN fullMd5 CHAR(24) GENERATED ALWAYS as (to_base64(unhex(md5(full)))) STORED NOT NULL;
```

Comment:
One thing I wonder about is how this will go with the Datasette export? It might (probably will?) break that, but then again I'm also not fully on top of how that export and re-import currently works, with Lars' recent changes there.

Well, we'll see 🫡

Reply:
I'll do a trial run on my local machine and test this before merging - nice catch to think about this :)